spec/smoke/api/v1/providers_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "spec_helper_smoke"
+
+describe "V1 Public API Smoke Tests", :aggregate_failures, smoke: true do
+  let(:course_year) { Time.current.year }
+  let(:base_url) { Settings.publish_url.sub("www", "api") }
+
+  subject(:response) { HTTParty.get(url) }
+
+  context "providers" do
+    describe "GET /v1/recruitment_cycles/:recruitment_year/providers" do
+      let(:url) { "#{base_url}/api/public/v1/recruitment_cycles/#{course_year}/providers" }
```

Comment:
```suggestion
      let(:url) { "#{base_url}/api/public/v1/recruitment_cycles/#{recruitment_year}/providers?page[per_page]=1" }
```

There is no such thing as `course_year`.
It probably going to be better to return 1 item, rather then the default 500.